### PR TITLE
Update jQuery version

### PR DESCRIPTION
**Pull request by mlxd**
**Context:** Recently, Dependabot issued an alert for docs/xanadu_theme/static/jquery.js and recommended an upgrade to a minimum jQuery version of 3.5.0. This PR addresses the security vulnerability and updates the instructions for building the Sphinx documentation.

**Description of the Change:** 
- The static copies of jQuery and Bootstrap in the CSS theme have been removed (these are pulled from a CDN instead).

**Benefits:** Fixes dependabot security alert

**Possible Drawbacks:** None

**Related GitHub Issues:** None. [ch7715]

### CODE REVIEW

File: .github/CHANGELOG.md [hunk 1]
```diff
@@ -4,17 +4,21 @@
 
 ### Improvements
 
+- Static versions of jQuery and Bootstrap are no longer included in the CSS theme. [(#)](https://github.com/XanaduAI/jet/pull/)
```
Comment (trbromley) [inline .github/CHANGELOG.md:7]:
```suggestion
- Static versions of jQuery and Bootstrap are no longer included in the CSS theme. [(118)](https://github.com/PennyLaneAI/pennylane-lightning/pull/118)
```

File: .github/CHANGELOG.md [hunk 2]
```diff
@@ -4,17 +4,21 @@
...
+
 - Move changelog to `.github` and add a changelog reminder.
 
 ### Breaking changes
 
 ### Bug fixes
 
+
+
```
Comment (trbromley) [inline .github/CHANGELOG.md:16]:
```suggestion
```

File: doc/xanadu_theme/layout.html [hunk 3]
```diff
@@ -223,6 +223,7 @@
 <!-- JQuery -->
 <script src="https://code.jquery.com/jquery-migrate-1.0.0.js"></script>
 <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
```
Comment (trbromley) [inline doc/xanadu_theme/layout.html:225]:
```suggestion
```
Need to remove the old one?

**Review by trbromley — COMMENTED** [review]
Thanks @mlxd!

**Comment by josh146** [comment]
image

Flyby comment purely to say that these are my favourite types of PRs 🙌

File: doc/xanadu_theme/layout.html (hunk 3 above)
Comment (mlxd) [reply doc/xanadu_theme/layout.html:225]: Ooops. Yep.

**Comment by co9olguy** [comment]
We should really backport this fix to any other repos that have the same legacy theme

**Comment by josh146** [comment]
> We should really backport this fix to any other repos that have the same legacy theme

Definitely time to move the theme to a central repository :(

**Review by trbromley — APPROVED** [review]

**Pull request merged**